drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/exporters/posting_export.rb

```diff
@@ -0,0 +1,67 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisExternalApis::AcHmis::Exporters
+  class PostingExport
+    attr_accessor :output
+
+    def initialize(output = StringIO.new)
+      require 'csv'
```

Comment:
What is the advantage of putting the `require` here vs at the top of the file?

Reply:
I'm not sure, this is a copy of a job that Todd created originally. I'd guess it lets you run the job in the rails console without doing an additional import